### PR TITLE
Commenting code

### PR DESCRIPTION
**Pull request by konstahakli**

### CODE REVIEW

**Review by JoonaPeltola — CHANGES_REQUESTED** [review]
Very good, but need more comments

File: LineFollower.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public void run() {
 				DEObj.setCMD(0);
 				DEObj.setLIV(2);
 			}
-
+			//Dodgin obstacle when sees it
```
Comment (Tuomas-Talkio) [inline LineFollower.java]:
```suggestion fix small typo
			//Dodging obstacle when sees it
```

**Review by JoonaPeltola — APPROVED** [review]
Now it is good 👍

**Review by Tuomas-Talkio — APPROVED** [review]
Good job, very good

**Pull request merged**